examples/flux_operator_ca_hpa/cluster-autoscaler/README.md

```diff
@@ -0,0 +1,71 @@
+# Setting Up Cluster Autoscaler in EKS
+Run this script to create a cluster and a nodegroup with 1 instance. The node group will have necessary tags required for cluster autoscaling. 
+
+```sh
+cd ..
+python3 create-delete-k8s-cluster.py --min-node-count 1 --max-node-count 3 --machine-type m5.large
+```
+
+### Create a service account for the kubernetes autoscaler. 
+
+First, create the IAM OIDC Provider for the cluster. Follow this link for more details - [Link](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html). Grab the cluster name from the output
+
+```bash
```

Comment:
No need to fix this here - but for your FYI - if you use "console" it looks really nice for output (like a pretty blue), and "bash" (or "sh") looks good for commands.